pkg/extender/extender.go

```diff
@@ -122,6 +122,9 @@ func (e *Extender) Stop() error {
 }
 
 func (e *Extender) serveHTTP(w http.ResponseWriter, req *http.Request) {
+	pvc1, err1 := core.Instance().GetPersistentVolumeClaim("restorePVC", "default")
```

Comment:
why is this code block needed here ?did you add it for debug purposes?

Reply:
Yes I did for debugging a UT. Good catch. Will remove this.